Build.ps1

```diff
@@ -0,0 +1,93 @@
+param(
+    [Parameter(Mandatory = $false)][string] $Configuration = "Release",
+    [Parameter(Mandatory = $false)][string] $VersionSuffix = "",
+    [Parameter(Mandatory = $false)][string] $OutputPath = "",
+    [Parameter(Mandatory = $false)][switch] $PatchVersion,
```

Comment:
In a later PR, you might want to look at [this PR](https://github.com/justeat/httpclient-interception/pull/4) to switch to using `Directory.build.targets` to generate the metadata so we don't need the extra steps in this file.

Reply:
I don't think we're really using that assembly versioning at all, just testing that it builds ok.
Remove `PatchVersion`?

Reply:
Hmm, how do we wire up "dotnet test" to that?

Reply:
I don't know how to do it for nunit. xunit has built-in detection for TeamCity and AppVeyor and pumps out the results appropriately so that those build systems discover the test output for themselves.